templates/product/_product_collections.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+{% load staticfiles %}
+{% load price_range from price_ranges %}
+{% load product_first_image from product_images %}
+
+<h2>See also</h2>
```

Comment:
It can't be collection name - product can be in many collections so we might display products from different collections.